src/components/tap-click/activator.ts

```diff
@@ -21,7 +21,7 @@ export class Activator {
     // queue to have this element activated
     this._queue.push(activatableEle);
 
-    rafFrames(2, () => {
+    rafFrames(6, () => {
```

Comment:
This has to be tested, but looks like 6 frames is the sweet spot to avoid activating an item while naturally scrolling.